server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestFunction.java

```diff
@@ -196,6 +197,12 @@ void diagnose_missing_function() {
                 new Range(),
                 "Expected a function name, but found 'FUNC1'",
                 DiagnosticSeverity.Error,
+                ErrorSource.PARSING.getText()),
```

Comment:
I suggest fixing the test cases instead, by changing the section type (in both cases).

Reply:
updated